### ISSUE TITLE
Configuration questions

### ISSUE THREAD

**Issue by ianhuynh** [mojodna/tessera#70]
Sorry for what could potentially a very obvious question.  I am new to tiles technology.  

I am able to get tessera up and running using  tessera -mbtiles://./my.mbtiles
How do I configure tessera so that it can serve up  a directory containing many *.mbtiles file? 

for example:   i have 2 mbtiles (  tile01.mbtiles and tile02.mbtiles) in folder //users/seth/
what is the correct way to setup config for this? 

I looked at the configuration below but don't quite understand how to setup tessera to serve them up.

````
{
  "/tile01": {
    "source": "mbtiles:///Users/seth/tile01.mbtiles",
    "cors": false,
    "timing": false
  },
  "/tile02": {
    "source": "mbtiles:///Users/seth/tile02.mbtiles",
    "cors": false,
    "timing": false
  },
 
}
````

**Comment by ianhuynh** [comment]
Correction:  using it looks like i can serve up multiple mbtiles.
{
  "/b": "mbtiles://./tiles-world-simple.mbtiles",
  "/foo": "mbtiles://./tiles-world-simple.mbtiles" , 
 "/APA" : "mbtiles://./APA.mbtiles"
}

however, it looks like mbtiles with format PNG can be served up in configuration file but not PBF mbtiles

this will render via config.json 
````
{
    "scheme": "tms",
    "basename": "tiles-world-simple.mbtiles",
    "id": "tiles-world-simple",
    "filesize": 787456,
    "bounds": [-180, -85.0511, 180, 85.0511],
    "center": [0, 0, 2],
    "minzoom": 0,
    "maxzoom": 4,
    "name": "tiles-world",
    "description": "",
    "legend": "",
    "template": "",
    "version": "1.0.0",
    "format": "png",
    "tiles": ["http://localhost:8080/b/{z}/{x}/{y}.png"],
    "tilejson": "2.0.0"
}
````

This won't render via config.json.  It only works using URI.

````
{
    "scheme": "tms",
    "basename": "APA.mbtiles",
    "id": "APA",
    "filesize": 80384000,
    "name": "APA",
    "description": "APA",
    "version": "2",
    "minzoom": 0,
    "maxzoom": 17,
    "center": [-97.249603, 34.303741, 17],
    "bounds": [-119.297018, 30.461132, -96.944396, 55.026472],
    "type": "overlay",
    "format": "pbf",
    "vector_layers": [{
        "id": "APA",
        "description": "",
        "minzoom": 0,
        "maxzoom": 17,
        "fields": {
            "Id": "Number",
            "OBJECTID": "Number",
            "Play": "String",
            "Shape_Area": "Number",
            "Shape_Leng": "Number",
            "Ticker": "String"
        }
    }],
    "tiles": ["http://localhost:8080/APA2/{z}/{x}/{y}.pbf"],
    "tilejson": "2.0.0"
}

````

**Comment by mojodna** [comment]
If using a URI in object form, it needs to (at least partially) match what `url.parse(uri, true)` would return. In practice, this means including `"protocol": "mbtiles", "host": ".", "pathname": "APA.mbtiles"`.

I can't remember if the rest of the TileJSON info (what's included there) can live under `query:` or if it belongs at the root. In practice, though, all of that info is provided by a `getInfo()` call which reads the metadata out of the MBTiles archive.

Tessera won't render PBFs to images (except using tilelive-xray) unless you have a companion `tmstyle` source that uses it and provides styling information.

I hope this helps!

**Comment by ianhuynh** [comment]
hmm.. https://github.com/mojodna/tessera/blob/master/server.js

when using opts.uri method, 
- line 41-43 loads the tile using the protocol declared by the URI
- then line 56-60 interrogates the format and it if is "pbf", it does some extra work to make sure that it serves the tile as xray.

when using opts.config method ( -c )
- line 97-99 does what 41-43 does above
- where is the equivalent for line 56-60 above? 

We were trying to provide preview of 'pbf' based tiles using -c option along with tilelive-xray.  And it didn't look like it was serving it up properly.    This also led us to propose a PR in server.js  : https://github.com/mojodna/tessera/pull/71/files/3fe8a04f5facd5d4431da14834aa98dada794227

Let me know if I am approaching this from the wrong direction.

**Comment by mojodna** [comment]
Yup, I understand what you were saying now. Thanks for finding and fixing this!
